lib/Core/Suggestion.php

```diff
@@ -2,8 +2,16 @@
 
 namespace Phpactor\Completion\Core;
 
+use RuntimeException;
+
 class Suggestion
 {
+    const TYPE_FUNCTION = 'f';
+    const TYPE_CLASS_MEMBER = 'm';
+    const TYPE_VARIABLE = 'v';
+    const TYPE_CONSTANT = 'm';
+    const TYPE_UNDEFINED = 'u';
```

Comment:
These are rougly mapping to VIM completion types, but we should just use verbose names such as `function`, `class_member` etc. and map to relevant client types in the client plugin,